Formula/latexindent.rb

```diff
@@ -1,8 +1,8 @@
 class Latexindent < Formula
   desc "Add indentation to LaTeX files"
-  homepage "https://ctan.org/pkg/latexindent"
-  url "https://github.com/cmhughes/latexindent.pl/archive/V3.17.3.tar.gz"
-  sha256 "8f7c4f3d98c5fff762db5d58367e3f2b48714dc0a21d6fda8a36ee8514dc66e1"
+  homepage "https://latexindentpl.readthedocs.io"
```

Comment:
Updated since the previous homepage was unreachable on linux (https://github.com/Homebrew/homebrew-core/pull/102996#issuecomment-1151259380, https://github.com/Homebrew/homebrew-core/pull/99304#issuecomment-1100932715)